packages/flutter/test/material/user_accounts_drawer_header_test.dart

```diff
@@ -216,6 +216,35 @@ void main() {
     expect(transformWidget.transform.getRotation()[4], 1.0);
   });
 
+  testWidgets('UserAccountsDrawerHeader icon color changes', (WidgetTester tester) async {
+    await tester.pumpWidget(MaterialApp(
+      home: Material(
+        child: UserAccountsDrawerHeader(
+              onDetailsPressed: () { },
+              accountName: const Text('name'),
+              accountEmail: const Text('email'),
```

Comment:
These 3 lines are indented too much.

Reply:
Agreed, should be 2 space indent

Reply:
I also agree, but this was the auto indentation given by IntelliJ and dartfmt IIRC, I will change them, as I don't like it as well.